pkg/exporter/process.go

```diff
@@ -125,6 +130,31 @@ func InitExportingProcess(input ExporterInput) (*ExportingProcess, error) {
 		pathMTU:         input.PathMTU,
 		templatesMap:    make(map[uint16]templateValue),
 		templateRefCh:   make(chan struct{}),
+		oneByte:         make([]byte, 1),
```

Comment:
>I don't want to recreate it every time when testing the connection. Moved it to ExportingProcess struct then.

This seems like a local variable required by a method rather than the struct. 
How about taking in this as an argument in `checkConnToCollector` and defining it along with the inline goroutine that is calling this method. If the argument is a pointer we could make sure it is only defined and allocated once.

Reply:
Makes sense to me. Updated. Thanks!